fastlane/metadata/en-US/app_store_screenshot_1.txt

```diff
@@ -0,0 +1,2 @@
+A simple,
```

Comment:
I don't think this comma makes sense

```suggestion
A simple
```

Reply:
Agreed, and with the forced new line, it looks weird too

```
A simple,
note taking experience
```

It was in the spec, though. I'll get rid of it here. Easy enough to restore anyways.